maro/cli/inspector/env_data_process.py

```diff
@@ -0,0 +1,201 @@
+import csv
+import json
+import os
+
+import numpy as np
+import pandas as pd
+import tqdm
+import yaml
+
+from maro.cli.inspector.launch_env_dashboard import launch_dashboard
+from maro.cli.inspector.params import GlobalFilePaths as Gfiles
+from maro.cli.inspector.params import GlobalScenarios
+from maro.utils.exception.cli_exception import CliException
+from maro.utils.logger import CliLogger
+
+logger = CliLogger(name=__name__)
+
+
+def start_vis(source: str, force: str, **kwargs):
+    """Entrance of data pre-processing.
+    Generate name_conversion CSV file & summary file.
+
+    Expected File Structure:
+    -input_file_folder_path
+        --epoch_0 : data of each epoch
+            --holder_info.csv: Attributes of current epoch
+        ………………
+        --epoch_{epoch_num-1}
+        --snapshot.manifest: record basic info like scenario name, name of index_name_mapping file
+        --index_name_mapping file: record the relationship between an index and its name.
+        type of this file varied between scenario.
+
+        summary file would be generated after data processing.
+
+    Args:
+        source(str): Data folder path.
+        force(str): expected input is yes/no. Indicates whether regenerate data.
```

Comment:
If the usage of force is what in the docsting, why the type is not bool?
Also, change the name to a more specific one.